### ISSUE TITLE
Warning when setting a boolean with cme modify

### ISSUE THREAD

**Issue by dod38fr** [dod38fr/config-model#19]
In `/etc/LCDd.conf`, set `ReportToSyslog=oui` (it's an invalid parameter).

Then run:
```
$ sudo cme modify lcdproc 'server ReportToSyslog=yes' -force
cme: using LCDd model
Argument "oui" isn't numeric in array or hash lookup at /usr/share/perl5/Config/Model/Value.pm line 1774.
Changes applied to lcdproc configuration:
- server ReportToSyslog: 'no' -> 'yes'
```

The warning does not help

**Comment by dod38fr** [comment]
This warning is due to the fact that `cme -force` uses `check => no` so bad values are stored in the configuration tree. This was done with #15 to avoid triggering an error while loading a license short_name that refer to a not-yet-existing license. See the comment at the end of this [mail](https://lists.debian.org/debian-devel/2017/10/msg00376.html)

Possible solutions:
* go back to cme -force means check => skip
* let user decide the check level (with -force (check: skip) and -force-more (check no) ??)
* allow storage if the value error is "only" a reference error, i.e handle check => skip as check => no (this would become a "special case" with all its drawback)
* implement a kind of transaction (way more complicated and probably very hard to get right...)

**Comment by dod38fr** [comment]
Solved in v2.124 with 890e9cdd9d101372118e1d3e73560f130efc0a46 (i got the wrong issue number in that commit)
